README.md

```diff
@@ -1,68 +1,73 @@
+
  <a href="https://discord.gg/jWVbPfC" target="_blank">
    <img src="https://img.shields.io/discord/308323056592486420?logo=discord"" alt="Discord Chat" />  
  </a>
 
-## Amplify for Android (Preview)
-
-AWS Amplify provides a high-level interface to perform different
-categories of cloud operations. Each category is fulfilled by a
-_plugin_. You specify which plugins to use during setup.
-
-The default plugins that we provide are designed to facilitate
-interaction with Amazon Web Services (AWS). But, the Amplify framework
-is designed to be extensible to any other backend or service.
-
-If you're new to the project, checkout the
-[Getting Started Guide](https://docs.amplify.aws/lib/getting-started/setup/q/platform/android).
-
-## The Categories
-
-### [DataStore](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/android):
-
-Model your app's data. Save, query, and observe changes to your data
-from a local repository. Let DataStore synchronize your local data with
-the Cloud. Our default implemenation syncs local data to/from an Amazon
-DynamoDB database, via an Amazon AppSync front-end.
-
-### [REST APIs](https://docs.amplify.aws/lib/restapi/getting-started/q/platform/android):
-
-Easy auth and request signing against multiple REST endpoints. Our
-default plugin works great with Amazon API Gateway.
-
-### [GraphQL APIs](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/android)
-
-Data modeling and simple auth against GraphQL endpoints. Our default
-plugin targets AppSync.
-
-### [Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android):
-
-Collect and report usage data for your app. Our default plugin
-communicates with Amazon Pinpoint.
-
-### [Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/android):
-
-Store and retrieve files in the Cloud. We use Amazon Simple Storage
-Service (S3) by default.
+AWS Amplify provides a high-level interface to perform different categories of
+cloud operations. Each category is fulfilled by a _plugin_. You specify which
+plugins to use during setup.
+
+The default plugins that we provide are designed to facilitate interaction with
+Amazon Web Services (AWS). But, the Amplify framework is designed to be
+extensible to any other backend or service.
+
+To familiarize yourself with Amplify, checkout our [Getting Started
+Guide](https://docs.amplify.aws/lib/getting-started/setup/q/platform/android).
+
+## Features / APIs
+
+- **[Analytics](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/android)**
+  Easily collect analytics data for your app. Analytics data includes user
+  sessions and other custom events that you want to track in your app.
+- **[API
+  (GraphQL)](https://docs.amplify.aws/lib/graphqlapi/getting-started/q/platform/android)**
+  Interact with your GraphQL server or AWS AppSync API with an easy-to-use &
+  configured GraphQL client.
```

Comment:
We should disband from unquantified phrases "simple" or "easy-to-use." To Amazon-ify such statements, we could corroborate the in terms of # of steps to get started -- or # of lines of code to get started, etc. The user can then decide for themselves whether or not it is easy (hopefully.)

```suggestion
  Designed for interaction with your GraphQL server or AWS AppSync API
```

Reply:
I copied all of this copy from the JavaScript README, with the thought that if we want to optimize we can do it across all three.

Reply:
Alright, cool. I don't love their descriptions, but there is a certain value in keeping the messaging consistent, I suppose.